src/main/java/com/microsoft/jenkins/appservice/commands/FTPDeployCommand.java

```diff
@@ -59,6 +62,9 @@ public void execute(final IFTPDeployCommandData context) {
             ftpUrl = ftpUrl.substring(0, splitIndex);
         }
 
+        // Stop app first to make sure all opening handlers released
+        context.getWebAppBase().stop();
```

Comment:
Check whether it's already stopped? If so no need to start.

Reply:
Updated.